arangodb-net-standard/Transport/Http/HttpApiTransport.cs

```diff
@@ -57,6 +64,14 @@ private static void ApplyHeaders(WebHeaderCollection webHeaderCollection, HttpHe
                     headers.Add(key, webHeaderCollection[key]);
                 }
             }
+            //build and add driver info header
+            string flags = string.Empty;
+            if (driverFlags != null)
+            {
+                flags = string.Join(";",driverFlags);
+            }
+            var assemblyInfo = Assembly.GetCallingAssembly().GetName();
+            headers.Add(CustomHttpHeaders.DriverInfoHeader, $"{assemblyInfo.Name}/{assemblyInfo.Version} ({flags})");
```

Comment:
@DiscoPYF , I agree with your suggestion about the constant string `DriverName`. Where can we place something like this?

Reply:
Looking at the existing classes we have, I would say either in `HttpApiTransport` or in `CustomHttpHeaders`. Or we could create a class for these sort of global constants, called `ArangoConstants` or `ApiConstants` or something of the sort. 🤔

Reply:
@DiscoPYF , thanks. I've created a class for global constants called `DriverConstants`. As a convention, we're using the project's github name for other drivers, so for this one it will be `arangodb-net-standard`